app/models/comment.rb

```diff
@@ -190,6 +193,12 @@ def calculate_score
     Comments::CalculateScoreWorker.perform_async(id)
   end
 
+  def update_cached_user
+    return unless user
+
+    self.cached_user = Articles::CachedEntity.from_object(user)
```

Comment:
`Articles::CachedEntity` seems like an odd name now that this is a cached User in use outside of Article!